draft-ietf-ppm-dap.md

```diff
@@ -165,8 +165,17 @@ aggregator.
 
 16:
 
-- Bump draft-irtf-cfrg-vdaf-13 to 14 {{!VDAF}} and adopt changes to the
-  ping-pong API (\#705, \#718).
+- Bump draft-irtf-cfrg-vdaf-13 to 15 {{!VDAF}} and adopt changes to the
+  ping-pong API. (\*) (#705, #718)
```

Comment:
"*" means wire breaking change. This isn't a wire breaking change, is it?

Reply:
You're right, I forgot that all the context strings or labels that incorporate draft versions are at the DAP layer.